src/components/RegistrationForm/Form/Form.jsx

```diff
@@ -1,5 +1,5 @@
-import {FormField} from './FormField.jsx';
-import {FormFieldPassword} from "./FormFieldPassword.jsx";
+import {FormField} from './fields/FormField.jsx';
+import {FormFieldPassword} from "./fields/FormFieldPassword.jsx";
```

Comment:
You could make an index.js and these 2 lines will become 1 line.

Reply:
created file /index.js for re-export components FormField & FormFieldPassword